hist/hist/src/TMultiDimFit.cxx

```diff
@@ -1962,8 +1967,10 @@ void TMultiDimFit::MakeRealCode(const char *filename,
    Int_t i, j;
 
    Bool_t  isMethod     = (classname[0] == '\0' ? kFALSE : kTRUE);
-   const char *prefix   = (isMethod ? Form("%s::", classname) : "");
-   const char *cv_qual  = (isMethod ? "" : "static ");
+   TString prefix;
+   const char *cv_qual  = isMethod ? "" : "static ";
+   if (isMethod)
+      prefix = TString::Format("%s::", classname);
```

Comment:
```suggestion
      prefix.Form("%s::", classname);
```
This does the formatting in place instead of creating a new `TString` and copying.

Reply:
Now here https://github.com/root-project/root/pull/13438/commits/7a9da27cbda6f5959c894edf8313354c498f5f3c